content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -376,7 +376,7 @@ For scaling down the stabilization window is _300_ seconds(or the value of the
 for scaling down which allows a 100% of the currently running replicas to be removed which
 means the scaling target can be scaled down to the minimum allowed replicas.
 For scaling up there is no stabilization window. When the metrics indicate that the target should be
-scaled up the target is scaled up immediately. There are 2 policies which. 4 pods or a 100% of the currently
+scaled up the target is scaled up immediately. There are 2 policies which are 4 pods or a 100% of the currently
```

Comment:
how about

```suggestion
scaled up the target is scaled up immediately. There are 2 policies where 4 pods or a 100% of the currently
```

Reply:
Fine by me @tengqm

Reply:
@juampynr you can click “Commit suggestion” to update with that change, or you can amend the existing commit.